ocs_ci/helpers/helpers.py

```diff
@@ -2898,6 +2898,8 @@ def storagecluster_independent_check():
         .get()
         .get("items")[0]
     )
+    if config.ENV_DATA["platform"].lower() in constants.MANAGED_SERVICE_PLATFORMS:
```

Comment:
Independent mode check  is needed for consumer cluster. I think the update is needed places where the selection of the storage class is done. 
Or something like this in constants.py
```
DEFAULT_CLUSTERNAME_EXTERNAL_MODE = "ocs-external-storagecluster" if config.ENV_DATA["platform"] not in MANAGED_SERVICE_PLATFORMS else DEFAULT_CLUSTERNAME
```

Reply:
That is a different topic: we don't have a good abstraction for storage class in ocs-ci.

Reply:
I agree with Jilju. This PR would invalidate the check and although there are differences in classic independent mode and managed service, we should keep the check as is to let us know that the cluster is in independent mode if the test is ran on consumer. We can handle differences between platforms at other places.